### ISSUE TITLE
New command line options to consider

### ISSUE THREAD

**Issue by derek-keeler** [eniklas/gamatrix-gog#37]
Things that we might want to affect via the command line but currently do not:

db_path: C:\Users\my_user\dev\gamatrix-gog-dbs
cache: C:\Users\my_user\dev\gamatrix-gog\my-cache-file
log_level: info
igdb_client_secret: abcdefghi01234jklmno56789pqrst
igdb_client_id: 0123uvwxyz4567abcde89012fg34hi

**Comment by eniklas** [comment]
Agreed. My original intention was that any option could be provided via the config file or command-line args, but once I started focusing on server mode, I neglected the CLI.

**Comment by eniklas** [comment]
Obsoleted by v2, closing.
